xarray/tests/test_cftimeindex_resample.py

```diff
@@ -110,22 +129,14 @@ def da(index) -> xr.DataArray:
     )
 
 
-@pytest.mark.parametrize("freqs", FREQS, ids=lambda x: "{}->{}".format(*x))
+@pytest.mark.parametrize(
+    "freqs", FREQS_WITH_TICK_RESAMPLE_FREQ, ids=lambda x: "{}->{}".format(*x)
+)
 @pytest.mark.parametrize("closed", [None, "left", "right"])
 @pytest.mark.parametrize("label", [None, "left", "right"])
 @pytest.mark.parametrize("offset", [None, "5s"], ids=lambda x: f"{x}")
-def test_resample(freqs, closed, label, offset) -> None:
+def test_resample_with_tick_resample_freq(freqs, closed, label, offset) -> None:
     initial_freq, resample_freq = freqs
-    if (
```

Comment:
Since our minimum version of pandas is greater than or equal to 2.2, I took the opportunity to remove this old test-skipping logic.